contributing/background/glossary.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+title: Glossary
+nav_order: 3
+parent: Background
+grand_parent: Contributing
+---
+# Glossary of Common Terms
+
+This page contains a selection of key terms important to understanding the structure of data within Data Commons.
+
+## Term List
+
+**[Statistical variable](https://datacommons.org/browser/StatisticalVariable)**
```

Comment:
I might propose adding the links separately (i.e. **Statistical variable ([link to node definition](...)) so that it's clearer and easier to make out. We might also want to add relative links to each of these terms, but I think that might require even more manual work.